scripts/welding_schema.ipynb

```diff
@@ -0,0 +1,485 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Welding schema"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%cd -q .."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# some python imports that will be used throughout the tutorial\n",
+    "import matplotlib.pyplot as plt\n",
+    "import networkx as nx\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pint\n",
+    "import sympy\n",
+    "import xarray as xr\n",
+    "from mpl_toolkits.mplot3d import Axes3D\n",
+    "\n",
+    "import asdf"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# importing the weldx package with prevalent default abbreviations\n",
+    "import weldx\n",
+    "import weldx.geometry as geo\n",
+    "import weldx.measurement as msm\n",
+    "import weldx.transformations as tf\n",
+    "import weldx.utility as ut\n",
+    "import weldx.visualization as vis\n",
+    "from weldx import Q_\n",
+    "from weldx.transformations import LocalCoordinateSystem as lcs\n",
+    "from weldx.transformations import WXRotation\n",
+    "from weldx.welding.groove.iso_9692_1 import get_groove"
```

Comment:
This is more a general comment since I noticed it while updating the tutorials: These are a lot of imports that might look a little bit intimidating for someone new (even though this is not a tutorial). However, I guess that most of them aren't used in this notebook anyway and they were just copied from other notebooks. ;) We should refactor the imports in the notebooks when we got the time

Reply:
yes this notebook is just for testing, we can delete this before releasing :)